config/environment.rb

```diff
@@ -48,7 +48,7 @@
 
   required(:aws).schema do
     # the region your AWS bucket is in
-    required(:region).filled(:str?)
+    # required(:region).filled(:str?)
```

Comment:
Why was this done?

Reply:
I reckon region is not strictly needed for S3 (and not for my setup anyway).